socorro/unittest/testbase.py

```diff
@@ -1,11 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-import unittest
-
-
-class TestCase(unittest.TestCase):
-
-    def shortDescription(self):
-        return None
```

Comment:
This `shortDescription` thing was to fix a thing nose did. We don't use nose anymore, so we can nix this file altogether.